### PR TITLE
C_CPP : Create STL_LISTS.md file 

### PR DESCRIPTION
**Pull request by Pranshu321**
<hr>

## Description 📜

In this i deeply explained about the list data structure which is defined in standard template library , in this i talked about the definition of list and completely explain the functions used in lists stl and proper code snippets and comments.
Issue closed : #1307

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/Templates & Libraries/STL LISTS.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+# STL LIST
+If we talk about the List in standard template library.
+<p> List is a sequence container that allows insertion and deletion operations in constant time from any place in the sequence and can iterate in both directions forward and backward but traversing is bit slower. </p>
+
```
Comment (vasu-1) [inline C_CPP/Templates & Libraries/STL LISTS.md:4]:
```suggestion

 - List is a sequence container that allows insertion and deletion operations in constant time from any place in the sequence and can iterate in both directions forward and backward but traversing is bit slower.
- It allows non-contiguous memory allocation.
```

Get rid of html tags in markdown !

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Great Job !

please do this changes then you are good to go ..

- change this file name to -> ` Stl_lists.md `
- after renaming this add the line  ` - [STL Lists](Stl_lists.md) ` into ` Templates & Libraries/readme.md `.
- Please add a code(implementation) and output for the same. (display some of the functions)

Kindly make this changes

File: C_CPP/Templates & Libraries/STL LISTS.md (hunk 1 above)
Comment (Pranshu321) [reply C_CPP/Templates & Libraries/STL LISTS.md]: sir i use p tag for making my content more readable and presentable and sir now you can check my work i add all the changes that you proposed . please check my work @vasu-1

**Review by vasu-1 — APPROVED** [review]
Good to go for Merge !!!!
@Saviour1001

File: C_CPP/Templates & Libraries/STL LISTS.md (hunk 1 above)
Comment (vasu-1) [reply C_CPP/Templates & Libraries/STL LISTS.md]: Cool 🥇

**Comment by Saviour1001** [comment]
Hey @Pranshu321 great work on the content ! As this is an STL Data Structure. Could you just place this content in the Data Structures Folder. You can place it [here](https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Data%20Structures) in a separate folder if you are willing to explain List more or in the STL.

**Comment by vasu-1** [comment]
> Hey @Pranshu321 great work on the content ! As this is an STL Data Structure. Could you just place this content in the Data Structures Folder. You can place it [here](https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Data%20Structures) in a separate folder if you are willing to explain List more or in the STL.

👍

**Comment by Pranshu321** [comment]
> Hey @Pranshu321 great work on the content ! As this is an STL Data Structure. Could you just place this content in the Data Structures Folder. You can place it [here](https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Data%20Structures) in a separate folder if you are willing to explain List more or in the STL. 

Then I also update that readme file in data structure file.

**Comment by Saviour1001** [comment]
yes, Remove the changes you did in this readme and add the new changes to the readme of the folder you will be finally placing it in.

**Comment by Pranshu321** [comment]
> yes, Remove the changes you did in this readme and add the new changes to the readme of the folder you will be finally placing it in.

I dont know how to delete the commit , please tell is their any way to delete commit by github desktop only.

**Comment by vasu-1** [comment]
@Saviour1001

**Comment by Pranshu321** [comment]
- Sir @Saviour1001  and @vasu-1 i am closing this PR because i am not able to delete my commit . I have a talk with @Saviour1001 sir.
- New merge new PR it have the same file that reviewed here but in correct folder.
- PR link : #2790

**Pull request closed without merging**